appservice/src/utils/random.ts

```diff
@@ -0,0 +1,16 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as crypto from "crypto";
+
+export namespace randomUtils {
+
+    export function getRandomHexString(length: number): string {
```

Comment:
I don't think this is used ever?

Reply:
The reason I have this in here is because the security board wants people to only ever use security-strong random number generators (not the built-in JS versions), so I'd like to encourage their use by having them in the library.  Functions extension is the only one that currently uses it, so I could pull it into there, but personally I'd rather have it available in the library.  Thoughts?

Reply:
I see your point about 'encouraging best practices', but this function feels unrelated to app service. I don't want this to turn into a dumping ground for assorted utility functions like the Uitilties project in CAT